docs/further-reading/fuzzer_environment.md

```diff
@@ -39,6 +39,14 @@ All build artifacts needed during fuzz target execution should be inside the
 Everything else is ignored (e.g. artifacts in `$WORK`, `$SRC`, etc) and hence
 is not available in the execution environment.
 
+We strongly recommend static linking because it just works. 
+However dynamic linking can work if shared objects are included in the `$OUT` directory and are loaded relative
+to `'$ORIGIN'`, the path of the binary (see the discussion of `'$ORIGIN'` [here](http://man7.org/linux/man-pages/man8/ld.so.8.html)).
+A fuzzer can be instructed to load libraries relative to `'$ORIGIN'` during compilation (e.g. `-Wl,-rpath,'$ORIGIN/lib'` )
+or afterwards using `chrpath` (e.g. `chrpath -r '$ORIGIN/lib' $OUT/$fuzzerName`). Note that `'$ORIGIN'` should be surronded
+by single quotes because it is not an environment variable like `$OUT` that can be retreived during execution of `build.sh`.
+Its value is retrieved during execution of the binary.
+
```

Comment:
Can you add a link to example build.sh (like facebook one).

Reply:
Facebook one is  a weird case because patchelf is called by `find`. Will do php which uses `chrpath`.